simple.js

```diff
@@ -496,6 +498,48 @@ define(function(require, exports, module) {
         }
 
         /**
+         * Detects Ace syntax mode from shebang for files with no extension
+         */
+        function detectSyntax() {
+            // syntax modes
+            var MODES = {
+                perl: /perl/,
+                php: /php/,
+                python: /python/,
+                ruby: /ruby/,
+                sh: /^(bash2?|ksh|sh)/
```

Comment:
Apparently vim [doesn't think](https://github.com/vim/vim/blob/071d4279d6ab81b7187b48f3a0fc61e587b6db6c/runtime/scripts.vim#L58-L67) they're the same? Can add anyway.